### PR TITLE
Allow for mysql configuration for SS

### PR DESCRIPTION
**Pull request by scollazo**
Related to https://github.com/archivematica/Issues/issues/931

### CODE REVIEW

**Comment by scollazo** [comment]
This is intended to be merged after #258

File: tasks/configure.yml [hunk 1]
```diff
@@ -5,15 +5,20 @@
...
   when: "archivematica_src_configure_ss_api_key is undefined and archivematica_src_configure_ss|bool"
 
-# Check whether SS user exists
-- name:  Check SS default test user
+- name: "Check for default SS user"
   become: "yes"
-  command: sqlite3  "{{archivematica_src_ss_environment['SS_DB_NAME']}}" "select username from auth_user where (id=1 and last_login is NULL and first_name='' and last_name='' and email='x@x.com' and username='test');"
+  shell: echo 'select username from auth_user where (id=1 and last_login is NULL and first_name="" and last_name="" and email="x@x.com" and username="test");'  | {{ archivematica_src_ss_virtualenv }}/bin/python manage.py dbshell
+  args:
+    chdir: "{{ archivematica_src_ss_app }}"
```
Comment (mamedin) [inline tasks/configure.yml:12]: Hi @scollazo we have an old typo on this variable:
```
maml@molinillo ~/external-roles/artefactual-archivematica-qa-src $ grep -Ri archviematica_src_ss_app *
tasks/configure.yml:    chdir: "{{ archviematica_src_ss_app }}"
tasks/configure.yml:    app_path: "{{ archviematica_src_ss_app }}"
tasks/ss-main.yml:      dest: "{{ archviematica_src_ss_app }}"
tasks/ss-main.yml:    app_path: "{{ archviematica_src_ss_app }}"
tasks/ss-db.yml:    app_path: "{{ archviematica_src_ss_app }}"
tasks/ss-db.yml:    app_path: "{{ archviematica_src_ss_app }}"
templates/etc/systemd/system/archivematica-storage-service.service.j2:WorkingDirectory={{ archviematica_src_ss_app }}
vars/main.yml:archviematica_src_ss_app: "/usr/lib/archivematica/storage-service"
``` 
So we can continue using the ` archviematica_src_ss_app` var or fix the typo.

**Review by mamedin — CHANGES_REQUESTED** [review]

File: tasks/configure.yml [hunk 2]
```diff
@@ -26,15 +31,21 @@
...
   when: archivematica_src_configure_ss|bool and ss_default_test_user.stdout != ""
 
-# Check whether SS user exists
-- name:  Check SS user
+- name: "Check for SS user"
   become: "yes"
-  command: sqlite3  "{{archivematica_src_ss_environment['SS_DB_NAME']}}" "select username from auth_user where username='{{ archivematica_src_configure_ss_user }}';"
+  shell: echo 'select username from auth_user where username="{{ archivematica_src_configure_ss_user }}";'  | {{ archivematica_src_ss_virtualenv }}/bin/python manage.py dbshell
+  args:
+    chdir: "{{ archivematica_src_ss_app }}"
```
Comment (mamedin) [inline tasks/configure.yml:38]: Same issue here

**Review by mamedin — APPROVED** [review]
LGTM!

**Pull request merged**